switchboard.json

```diff
@@ -0,0 +1,162 @@
+    "package_name": "phantom_switchboard",
+    "main_module": "switchboard_connector.py",
+    "min_phantom_version": "4.9.0",
+    "app_wizard_version": "1.0.0",
+    "contributors": [
+        {
+            "name": "Mhike"
+        }
+    ],
+    "configuration": {
+        "https_port": {
+            "description": "Splunk SOAR HTTPS port if your instance uses one other than 443",
+            "data_type": "string",
+            "order": 0,
+            "name": "https_port",
+            "id": 0
+        },
+        "repository_filter": {
+            "description": "A comma seperated list of repositories that shouldn't be added to playbook cache",
+            "data_type": "string",
+            "order": 1,
+            "name": "repository_filter",
+            "id": 1
+        },
+        "cache_expiration": {
+            "description": "How long should the playbook cache be valid in seconds",
+            "data_type": "numeric",
+            "required": true,
+            "default": "600",
+            "order": 2,
+            "name": "cache_expiration",
+            "id": 2
+        },
+        "debug": {
+            "description": "Print debugging statements to log",
+            "data_type": "boolean",
+            "default": "False",
+            "order": 3,
+            "name": "debug",
+            "id": 3
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate connectivity to the Splunk SOAR instance",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "run playbooks",
+            "identifier": "run_playbooks",
+            "description": "Identify and execute matching playbooks",
+            "verbose": "Based on the standard naming conventions used by the switchboard app, run all matching playbooks in the playbook cache.",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "repository_name": {
+                    "description": "The name of the repository to execute playbooks from",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0,
+                    "name": "repository_name"
+                },
+                "product_name": {
+                    "description": "Product strings to match on",
+                    "data_type": "string",
+                    "order": 1,
+                    "name": "product_name"
+                },
+                "rule_name": {
+                    "description": "Rule name to match on",
+                    "data_type": "string",
+                    "order": 2,
+                    "name": "rule_name"
+                }
+            },
+            "output": [
```

Comment:
Would be good to return some data in columns or custom table if appropriate

Reply:
The executed playbooks will show in the activity pane so I'm not sure if there is much value in rendering a widget since those are visible in the same UI screen. I could potentially see value if people were concerned about seeing directly which playbooks were interacted with via switchboard outside of the playbook itself but I haven't encountered anyone interested in that yet.

Reply:
Ok I'll leave the decision up to you, if you'd like to add some output later on